factory/post.js

```diff
@@ -26,7 +25,10 @@ function processContent(data) {
     var item = entries[i],
         postTitle = item.title.$t,
         content   = item.content.$t,
-        url       = item.link[4].href,
+        // Do not assume the link will always remain item 4 in the array
+        url       = _.filter(item.link, function (link) {
+          return link.rel === 'alternate' && link.type === 'text/html';
+        })[0].href,
```

Comment:
Take a look at the raw JSON data. There are 4 link objects in an array. The assumption is that the link we need is at position 4. This is not guaranteed to always be correct, since Google can choose to add a new link tomorrow at the 0th position, or decide to change the order, or anything like that. 

What this code does is filter the array to find the particular link we're looking for. The link we're looking for is of rel=alternate, and type=text/html (we have to assume that Google to decide to also add a new alternate link, say text/plain).

After filtering, only one item is guaranteed to remain in the array, and that is the element with rel=alternate and type=text/html. We simply retrieve this element with [0].